.travis.yml

```diff
@@ -0,0 +1,54 @@
+language: android
+sudo: required
+jdk: oraclejdk8
```

Comment:
Is it possible to switch to openjdk for Android?

Reply:
Yes, it is. I've just switched to openjdk